src/server/zset_family_test.cc

```diff
@@ -523,6 +523,16 @@ TEST_F(ZSetFamilyTest, ZInterStore) {
   EXPECT_THAT(resp.GetVec(), ElementsAre("b", "3"));
 }
 
+TEST_F(ZSetFamilyTest, ZInter) {
+  EXPECT_EQ(2, CheckedInt({"zadd", "z1", "1", "one", "2", "two"}));
+  EXPECT_EQ(3, CheckedInt({"zadd", "z2", "1", "one", "2", "two", "3", "three"}));
+  RespExpr resp;
+
+  resp = Run({"zinter", "2", "z1", "z2"});
+  EXPECT_THAT(resp, ArrLen(2));
+  EXPECT_THAT(resp.GetVec(), UnorderedElementsAre("one", "two"));
```

Comment:
Please do remove the `Unordered` part, as mentioned above :pray:

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2164/commits/3085a90b7b62c6ed38a43f351674ad88efbd58a0